tests/integration/query_engines/test_mssql.py

```diff
@@ -68,8 +68,17 @@ def wrap_select_queries():
     mocked = mock.Mock()
 
     def wrapper(self, *args, **kwargs):
-        if args and isinstance(args[0], Select):
-            mocked(*args, **kwargs)
+        if args and isinstance(args[0], sqlalchemy.sql.Select):
+            try:
+                mocked(*args, **kwargs)
+            except Exception:
+                # Force the database connection to revalidate. Ideally we would be able
```

Comment:
"revalidate" doesn't quite scan right for me here, am I missing something?

Reply:
Oh yes, fair point. I used that term because I'd been knee-deep in SQLAlchemy's internals for a while and the effect of nulling this value out is to force SQLAlchemy to call a `revalidate_connection()` method which ends up raising the error we were seeing.

I think the comment ought to be:
> Simulate the effect of getting an error from the underlying database driver.